doc/api/path.md

```diff
@@ -267,6 +267,38 @@ path.format({
 // Returns: 'C:\\path\\dir\\file.txt'
 ```
 
+## `path.fromURL(pathOrUrl)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `pathOrUrl` {string|URL}.
+* Returns: {string}
+
+Converts a [`URL`][] instance to a path string,
+returns `pathOrUrl` if it is already a string.
+
+A [`TypeError`][] is thrown if `pathOrUrl` is a
+[`URL`][] with a schema other than `file://`.
+
+```js
+path.fromURL(new URL('file:///Users/node/dev'));
+// Returns: '/Users/node/dev'
+
+path.fromURL('file:///Users/node/dev');
+// Returns: 'file:///Users/node/dev'
+
+path.fromURL(new URL('file:///c:/foo.txt'));
+// Returns On Windows: 'c:\\foo.txt'
+
+path.fromURL('index/path');
+// Returns: 'index/path'
```

Comment:
Does it return `index/path` on Windows, too?

Reply:
yes, non `URL` instances (i.e) strings just passthrough